plugin/trino-redis/src/main/java/io/trino/plugin/redis/RedisConnectorConfig.java

```diff
@@ -117,6 +119,20 @@ public RedisConnectorConfig setRedisScanCount(int redisScanCount)
         return this;
     }
 
+    @Min(1)
+    public int getRedisGetKeySize()
+    {
+        return redisGetKeySize;
+    }
+
+    @Config("redis.get-key-size")
```

Comment:
Maybe, `redis.max-keys-per-scan`, `redis.max-keys-per-fetch` or something is better.